rest/system-browse-get.rst

```diff
@@ -2,10 +2,9 @@ GET /rest/system/browse
 ===================
 
 Returns a list of directories matching the path given by the optional parameter
-``current``. The path can use `patterns as described in go's filepath package
-<https://golang.org/pkg/path/filepath/#Match>`_ and a `*` will be suffixed. It
-is also possible to give a path to a directory to list its contents by adding a
-trailing path separator.
+``current``. The path can use `patterns as described in Go's filepath package
+<https://golang.org/pkg/path/filepath/#Match>`_. It is also possible to list a
+directory's contents by adding a trailing path separator.
```

Comment:
I don't think the asterisk bit is insignificant. According to the linked patterns `foo` does not match `foobar` but  for this API call it does.